spec/services/search/alert_builder_spec.rb

```diff
@@ -1,4 +1,5 @@
 require "rails_helper"
+include DistanceHelper
```

Comment:
as above

Reply:
Isn't this used on line 13?

Reply:
it's already included in `spec/rails_helper.rb`